conjur/login/credential_data.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+"""
+CredentialData module
```

Comment:
Doesn't credentialsData used outside of `login`?
If so take it out of this folder, since it doesn't belong here.

Reply:
yes it's used in Logout. I will move it